src/bullets/data_source/data_source_fmp.py

```diff
@@ -1,8 +1,12 @@
 import json
-import math
-from datetime import datetime, date, timedelta
-from bullets.data_source.data_source_interface import DataSourceInterface, Resolution
+from datetime import date, timedelta
+
+from bullets.utils.market_utils import is_market_open, get_date_in_x_market_days_away, get_moments
+
```

Comment:
You can also remove the blank lines here

Reply:
Changed